datastore/src/main/java/com/ds/datastore/BookStoreController.java

```diff
@@ -144,10 +141,13 @@ protected void addServer(@RequestBody String json, @PathVariable String id) {
     }
 
     @GetMapping("/bookstores")
-    protected CollectionModel<EntityModel<BookStore>> all() throws Exception {
+    protected CollectionModel<EntityModel<BookStore>> all(@RequestParam(required = false) List<String> id) throws Exception {
```

Comment:
I cant recall - this list works with `RequestParam` if you specify the `id` multiple times?

Reply:
Yes, it works